.plop/templates/reactComponent.tsx.hbs

```diff
@@ -7,7 +7,10 @@ import * as S from "./styles";
 {{/if}}
 import { IProps } from "./types";
 
-export const {{> p_componentName}}: React.FC<IProps> = ({
+/**
+ * Example component description.
```

Comment:
There is no point in keeping autogenerated comments in repo.

Reply:
There are multiple ones like those.

Reply:
ok, removed